build/NuSpecs/Microsoft.ProjectReunion.nuspec

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+     <metadata minClientVersion="3.3">
+         <id>Microsoft.ProjectReunion</id>
+         <version>1.0</version>
+         <title>$id$</title>
+         <authors>Microsoft</authors>
+         <description>Project Reunion</description>
```

Comment:
>Project Reunion [](start = 22, length = 15)

doesn't sound very helpful :). we can change later though

Reply:
Sure.  I just copied what it said in the original ProjectReunion nuspec file.  I'm sure a PM will come up with something :)